test

```diff
@@ -1,12 +1,7 @@
 #! /usr/bin/env bash
 #
-# Run the semgrep-core test program from the correct folder.
-# This is meant for manual use, when we want to select which tests to run.
+# This calls the test program generated by alcotest_ext.
+# We're not using a symlink because dune rejects dangling symlinks.
```

Comment:
really? but bin/ is a dangling symlink now and dune is fine with it.

Reply:
oh, that's great. It used to be a problem (possibly https://github.com/ocaml/dune/issues/3819)